package.json

```diff
@@ -446,6 +446,13 @@
         "title": "View Flink Statement Results",
         "category": "Confluent: Flink Statements"
       },
+      {
+        "command": "confluent.handleUri",
+        "title": "Handle URI",
+        "icon": "$(link)",
+        "category": "Confluent",
+        "enablement": "confluent.e2eTesting"
```

Comment:
@flippingbits good suggestion! I've added this new context value and confirmed https://github.com/confluentinc/vscode/pull/2585 is still able to use it. (Note [`enablement` uses `when` clauses](https://code.visualstudio.com/api/references/contribution-points#contributes.commands))